### DISCUSSION TITLE
Pricing Plan & Portability..

### DISCUSSION THREAD

**Discussion by bipharish** [Budibase/budibase#1475] [Help] [answered]
Could you please tell me what this pricing mean? 5$/Month/End-user mean?

Say, I have 1000 users or customers or trial members, should I have to pay 1000*5$/Month? Or is the end-user mean me and those from my team need to pay 5$/Month? 

This has the potential have to change the pricing significantly ($5000 vs $50) and unusable for most startups. Please clarify this.

Also would like to know what is global end-user management, global builder management? could not find any documentation for it.

![image](https://user-images.githubusercontent.com/83997538/117752662-cd756b00-b234-11eb-81a0-98ddd0b93697.png)

**Comment by joebudi** [comment, marked as answer]
hi @Harish-2 . 

> Say, I have 1000 users or customers or trial members, should I have to pay 1000*5$/Month? Or is the end-user mean me and those from my team need to pay 5$/Month?

It does indeed mean you pay 1000 * $5 ($5000) if you decide to go with our standard plan.

You can use Budibase for free or course - it is open source!

But, if you want premium features, you would need to pay for them. Global end-user mgmt and global builder mgmt are methods of managing your users/employees and your builder. Basically, this functionality gives you more control of how people build with Budibase within your instance, and how to manage the end-users of your apps. 

Ultimately, the pricing is not confirmed, and not yet released. We may include global end-user mgmt and global builder mgmt in the open-source product.

I hope that helps.

**Reply by bipharish** [reply]
Thanks for clarification. What functionality will I have in opensource and what will be in the standard plan is still unclear to me. Why? because I don't know what "global end-user mgmt and global builder mgmt" mean and would I ever need it? It would be good to know about it in the near future if not now.

Yes, Budibase is open source which is great. In fact, it is amazing indeed. However, I have an important question.

Will we have access to the source of the project we create or we have only access to the docket image? And if for whatever reason, I want to move my project out of budibase to a different platform, is it possible? If yes, how could this be done? i.e. how to get the source code of the project developed by us and in which format will they be available?

**Reply by mjashanks** [reply]
Hi There. 

Firstly, I'd like to add some additional context. Budibase is designed for building applications to be run INSIDE a business - i.e. operational applications. So, the person who is building the applications is usually working for the same company as those using the applications.

So, our premium features will be targeted at managing larger deployments. E.g. If you are running using Budibase to build apps for your 50 employee business - you'll likely be fine with the open-source version. However, as companies start to get larger, they need, for example, to group their users into groups e.g. "Finance Team" and "Marketing Team" - and apply security roles for each group. This is the kind of thing that would be included in "Global User Management".

In terms of exporting the code, the answer is no. But I am interested in why you need to? I can think of a few possible reasons:
1. You want to take your app(s) and run them in your cloud of choice. You can already do this - the product is open source.
2. You want some extra flexibility - e.g. add code of your own. Over the next few months, we will be adding the ability to extend budibase with code (frontend and backend). Is this something that interests you?

Are there any other reasons for you needing to export the code? 

It doesn't make a whole lot of sense for Budibase apps, as Budibase is a Full-stack platform. This means that the Front end always expects the backend to be there - i.e. it's always making specific requests to the Budibase API - so breaking it apart would mean you have to redevelop the Budibase backend, or swap out all the API requests.

**Reply by joebudi** [reply]
Cheers @mjashanks

**Reply by bipharish** [reply]
Thanks for a detailed reply. I will review this with my team and get back shortly.

**Reply by fatherofinvention** [reply]
I have to say, the professionalism and thoroughness from the Budibase team when fielding questions here and on Discourse is super impressive. I'm really excited to see what comes on May 25th. Keep up the great work.
